core/src/main/java/io/syndesis/core/KeyGenerator.java

```diff
@@ -15,6 +15,8 @@
  */
 package io.syndesis.core;
 
+import net.iharder.Base64;
```

Comment:
There is an implementation of ordered base64 in `spring-security-crypto`[1], not sure if it makes much difference but would be a known dependency to us.

[1] https://docs.spring.io/autorepo/docs/spring-security/4.2.3.RELEASE/apidocs/org/springframework/security/crypto/codec/Base64.html#ORDERED

Reply:
That doesn't do what we want unfortunately... Just tried to migrate across and only private, internal methods do what we want.